README.md

```diff
@@ -147,7 +147,116 @@ You can also use Docker Compose to run all the components of EvalAI together. Th
     ```
     python scripts/workers/submission_worker.py
     ```
+    
+## Installation Instructions for Windows
 
+Setting up EvalAI on your local machine is really easy.
+Follow this guide to setup your development machine.
+
+1. Install [python] 2.x (EvalAI only supports python2.x for now.), [git], [postgresql] version >= 9.4, [RabbitMQ] and [virtualenv], in your computer, if you don't have it already.
+*If you are having trouble with postgresql on Windows check this link [postgresqlhelp].*
+
+2. Get the source code on your machine via git.
+
+    ```
+    git clone https://github.com/Cloud-CV/EvalAI.git evalai
+    ```
+
+3. Create a python virtual environment and install python dependencies.
+
+    ```
+    cd evalai
+    virtualenv venv
+    cd venv/Scripts
+    activate.bat    # run this command everytime before working on project
```

Comment:
Ummm... I am not sure what this is for. Can you please provide me a reference link?

Reply:
This is for createing a python virtual environment and install python dependencies.